pkgs/tools/games/scarab/default.nix

```diff
@@ -1,13 +1,10 @@
 {
   lib,
+  bc,
   buildDotnetModule,
   fetchFromGitHub,
-  glibc,
```

Comment:
Removing these is good if they aren't needed, but I think it should be a separate commit from the wrapper, in case we need to bisect/revert.